packages/components/src/components/gif.tsx

```diff
@@ -19,6 +19,8 @@ export const getColor = () => GRID_COLORS[Math.round(Math.random() * (GRID_COLOR
 
 const hoverTimeoutDelay = 200
 
+const Container = (props: any) => (props.href ? <a href={props.href} {...props} /> : <div {...props} />)
```

Comment:
@giannif wasn't sure what to do for preact since there is no `HTMLProps` type. I left it as `any` for now, but can definitely update it to whatever